scripts/make_fake_fixtures.py

```diff
@@ -21,92 +42,145 @@
     tournaments.append({
-        'pk': 'T%s' % str(tourney_id),
+        'pk': tourney_id,
         'model': 'agagd_core.tournament',
         'fields': {
             'total_players': random.randint(4,20),
-            'city': '',
-            'elab_date': '2013-07-01',
-            'description': '',
-            'wall_list': '',
-            'state': '',
+            'city': random.choice(CITY_NAMES),
+            'elab_date': elab_date.strftime("%Y-%m-%d"),
+            'description': random_state + tourney_id,
+            'wall_list': "1: Mal, Bruce                  2d     2+/w0     3+/w0     4+/w0  3-0-0\n"
+                         "2: Lee, Parker                 1d     1-/b2     4+/w0     3-/w0  1-2-0\n"
+                         "3: Lee, Matt                   1k     4-/w0     1-/b6     2+/b4  1-2-0\n"
+                         "4: Frye, Sam                   3k     3+/b2     2-/b6     1-/b8  1-2-0\n"
+                         "Note: This  is not generated by the AGAGD.",
+            'state': random_state,
             'rounds': random.randint(2,5),
-            'tournament_date': '2013-07-01'
+            'tournament_date': date.strftime("%Y-%m-%d")
         }
     })
 
 games = []
-for game_id in range(game_count):
-    p1 = random.choice(members)['pk']
-    p2 = random.choice(filter(lambda m: m['pk'] != p1, members))['pk']
+for game_id in range(1, game_count+1):
+    p1 = random.choice(member_ids)
+    p2 = random.choice([member_id for member_id in member_ids if member_id != p1])
+    color_1 = random.choice(['B', 'W'])
+    color_2 = 'B' if color_1 != 'B' else 'W'
     date = dt.date.today() - dt.timedelta(days = random.randint(2,20))
+    elab_date = date + dt.timedelta(days = 7)
     games.append({
         'pk': game_id,
         'model': 'agagd_core.game',
         'fields': {
             'pin_player_2': p2,
             'tournament_code': random.choice(tournaments)['pk'],
-            'rated': '',
-            'elab_date': date.strftime("%Y-%m-%d"),
-            'handicap': random.randint(0, 2),
-            'online': '',
-            'color_2': '',
+            'rated': random.randint(0, 1),
+            'elab_date': elab_date.strftime("%Y-%m-%d"),
+            'handicap': random.randint(0, 9),
+            'online': random.randint(0, 1),
+            'color_2': color_2,
             'sgf_code': '',
-            'komi': 0.0,
+            'komi': random.randint(0, 9),
             'pin_player_1': p1,
             'rank_1': '',
-            'result': '',
+            'result': random.choice(['B', 'W']),
             'rank_2': '',
             'game_date': date.strftime("%Y-%m-%d"),
-            'exclude': '',
-            'round': '',
-            'color_1': ''
+            'exclude': random.randint(0,1),
+            'round': random.randint(2,5),
+            'color_1': color_1
         }
     })
 
 chapters = [] 
-for i, chap_code in enumerate(CHAPTER_CODES):
+for member_id in range(0, len(CHAPTER_CODES)):
```

Comment:
I have realized that this will create data that does not conform to the way our prod database is set up. In practice, all of the values in `Chapters.member_id` correspond to a matching `Members.member_id`. (`Chapters.member_id` is both a primary key and a foreign key) This test data has chapter IDs that overlap with real, non-chapter Member objects.

This is good enough for *most* testing purposes, but it doesn't match how the schema is designed to be used. Do you think it's worth correcting now?